pydis_site/apps/content/resources/guides/pydis-guides/how-to-contribute-a-page.md

```diff
@@ -22,17 +22,38 @@ As website changes require staff approval, discussing the page content beforehan
 
 ## Creating the Page
 All pages are located in the `site` repo, at the path `pydis_site/apps/content/resources/`. This is the root folder, which corresponds to the URL `www.pythondiscord.com/pages/`.
+
 For example, the file `pydis_site/apps/content/resources/hello-world.md` will result in a page available at `www.pythondiscord.com/pages/hello-world`.
 
+#### Page Categories
 Nested folders represent page categories on the website. Each folder under the root folder must include a `_info.yml` file with the following:
 
 ```yml
-name: Category name
+title: Category name
 description: Category description
+icon: fas fa-folder # Optional
 ```
 
 All the markdown files in this folder will then be under this category.
 
+#### Having the Category also be a Page
+In order to make categories a page, place a page inside the category folder **with the same name as the category folder**.
+
+```plaintext
+guides
+├── contributing
+│   ├── _info.yml
+│   ├── contributing.md
+│   └── bot.md
+└── _info.yml
+```
+
+In the above example, `www.pythondiscord.com/guides/` will list `Contributing` as a category entry with information from `contributing/_info.yml`.
+
+However, `www.pythondiscord.com/guides/contributing` will render `contributing.md` rather than show the category contents, so *it is the article's responsibility to link to any subpages under the article*.
```

Comment:
Have you considered any way to show the direct children, or even all children? Perhaps a dropdown menu somewhere, like near the breadcrumbs.

Reply:
I have. Since I'm just trying to get feature parity with the current site, I'll open a separate issue for this so the app can move along.

Reply:
Plot twist: I did it anyways since it's it's not a big change. The dropdown only shows up on category-page-hybrids and look and function similar to the one shown on the current site when you're logged in.

The differences are that it only shows direct children, since any sub-categories show their children on their page anyway, and that the dropdown is click-toggled and not hover-toggled to be mobile-friendly.